web/book/src/language-features/README.md

```diff
@@ -3,16 +3,27 @@
 The pages of this section describe how PRQL handles various aspects of the
 language.
 
-| Feature          | Purpose                                                                 |
-| ---------------- | ----------------------------------------------------------------------- |
-| Coalesce         | [Handle potentially NULL values](./coalesce.md)                         |
-| Dates & times    | [Handle dates and times](./dates-and-times.md)                          |
-| Distinct         | [Equivalent of SQL `DISTINCT`](./distinct.md)                           |
-| Null handling    | [Handle `NULL` values](./null.md)                                       |
-| Ranges           | [Syntax for all forms of ranges](./ranges.md)                           |
-| Regex            | [Handle regular expressions](./regex.md)                                |
-| Strings          | [Rules for creating strings](./strings.md)                              |
-| S-strings        | [Insert SQL directly into a query with an S-string](./s-strings.md)     |
-| F-strings        | [Combine several column's data with F-strings](./f-strings.md)          |
-| Case             | [Create a new column based on the contents of other columns](./case.md) |
-| Target & Version | [Specify a target SQL engine and PRQL version](./target.md)             |
+<!--
+To work around https://github.com/rust-lang/mdBook/issues/984 we link to "index.html" instead of "README.md".
+To avoid complaints from `markdown-link-check` in the CI, we use "markdown-link-check-disable"
+-->
```

Comment:
I think the comment should be moved down next to the link-check-disable comment. And I don't think elaborating on the markdown-link-check-disable comment is necessary.